docs/Guide/utilities/time-utilities.mdx

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 5
+title: Using Time Utilities
+---
+
+## Introduction
+
+Often working with `Date` related objects can be hard to handle. `@sapphire/time-utilities` is a small package that
+contains a few utility classes that eases working with these kind of objects.
+
+## Installation
+
+```bash npm2yarn2pnpm
+npm install @sapphire/time-utilities
+```
+
+## Enums
+
+TODO: https://www.sapphirejs.dev/docs/Documentation/api-utilities/enums/time_utilities_src.Time
```

Comment:
Can you fill out this TODO I put into here while changing wording here and there?

Reply:
Yes 😃